popup.html

```diff
@@ -5,7 +5,9 @@
       </style>
     </head>
     <body>
-      <input type="checkbox" id="active"/> Activate timer
+      <label for="reload-int-input">Refresh interval (seconds)</label>
+      <input type="number" min='1' id="reload-int-input" />
```

Comment:
As I mentioned in another comment on the PR, we could add a placeholder to talk about default value or straightaway set the default in the input when there is nothing!

Reply:
Also do you think we could show the current defined reload interval to the user?

Reply:
Done. The default is set to 10 seconds, and it's displayed to the user if they open the popup while the auto-reload function is running.